example/example_source/lib_depending_on_lib_with_public_kokkos_dependency/fupp.cpp

```diff
@@ -0,0 +1,25 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <iostream>
+
+void print_fuzz();
+
+void print_full() {
```

Comment:
Should it be called `fupp`?

Reply:
yes, good catch, done in https://github.com/kokkos/kokkos/pull/7582/commits/18dc786d1d892f77f2364ef7dc706067509e7155